build.gradle

```diff
@@ -4,9 +4,6 @@ plugins {
     id 'jacoco'
 }
 
-sourceCompatibility = JavaVersion.VERSION_13
-targetCompatibility = JavaVersion.VERSION_13
-
```

Comment:
Struggling to get intellij to run tests with these lines present. 

Initially had an error message referring to an incompatible jdk. Altered some settings to get past this but now the test runner just doesn't discover the tests at all. Running the gradle test runner works.

Reply:
What happens if you drop to 12? Or even 11?

Reply:
12 works. Which I don't entirely understand given that, as far as I can tell I've specified in my IntelliJ preferences that jvm 13.0.1 should be used.